### PR TITLE
DOC: Improve docstring of Timedelta.delta

### PR DESCRIPTION
**Pull request by chalmerlowe**
- [ ] closes #xxxx
- [ ] tests added / passed
- [X] passes `git diff upstream/master -u -- "*.pyx" | flake8 --diff`
- [ ] whatsnew entry

```
################################################################################
###################### Docstring (pandas.Timedelta.delta) ######################
################################################################################

Return the timedelta in nanoseconds (ns), for internal compatibility.

Returns
-------
int
    Timedelta in nanoseconds.

Examples
--------
**Using string input**

>>> td = pd.Timedelta('1 days 42 ns')
>>> td.delta
86400000000042

>>> td = pd.Timedelta('3 s')
>>> td.delta
3000000000

>>> td = pd.Timedelta('3 ms 5 us')
>>> td.delta
3005000

**Using integer input**

>>> td = pd.Timedelta(42, unit='ns')
>>> td.delta
42

################################################################################
################################## Validation ##################################
################################################################################

Errors found:
	No extended summary found
	See Also section not found
```

### CODE REVIEW

File: pandas/_libs/tslibs/timedeltas.pyx [hunk 1]
```diff
@@ -770,7 +799,48 @@ cdef class _Timedelta(timedelta):
 
     @property
     def resolution(self):
-        """ return a string representing the lowest resolution that we have """
+        """
```
Comment (WillAyd) [inline pandas/_libs/tslibs/timedeltas.pyx]: Isn't this already a part of #21122?

**Review by WillAyd — CHANGES_REQUESTED** [review]

File: pandas/_libs/tslibs/timedeltas.pyx (hunk 1 above)
Comment (chalmerlowe) [reply pandas/_libs/tslibs/timedeltas.pyx]: @WillAyd 

Yes...
I am not sure why that is in there.
I created separate branches for each of the changes I am proposing to:

* delta
* resolution
* nanoseconds

etcetera, but the commit stream above seems to be sucking all of them into this PR.
super confused.

Comment (WillAyd) [reply pandas/_libs/tslibs/timedeltas.pyx]: Hmm this should just reflect what you have locally - any chance you merged your branches there accidentally?

Comment (jreback) [reply pandas/_libs/tslibs/timedeltas.pyx]: when creating a new branch, do it off of upstream/master (or if you keep a local master then off of this). each branch should be independent.

you will need to revert the resolution part of this change (as its in the other PR). or can just close that one and do it here is ok too)

**Review by jreback — CHANGES_REQUESTED** [review]

**Comment by chalmerlowe** [comment]
i did some looking at how to `git` rid of (pun intended) a bunch of the changes that are cluttering up the works.
think I am gonna just close this, try to submit a new pull request starting with a clean slate.
See PR #21138

**Pull request closed without merging**